rfcs/0098-community-team.md

```diff
@@ -0,0 +1,194 @@
+---
+feature: community-team
+start-date: 2021-07-31
+author: Irenes (Irene Knapp)
+co-authors: ashkitten
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Establish an official community team to make moderation decisions, model social norms, and mediate interpersonal problems.
+
+# Motivation
+[motivation]: #motivation
+
+There is not currently any official mechanism for moderation action. It's not sustainable to have to call on Graham any time there's a spammer or conflict that requires moderation, and we'd like to help the community become more self-regulating.
+
+# Goals
+[goals]: #goals
+
+* Model and enforce social norms
+* Mediate disputes when doing so is beneficial to the community
+* Respect the privacy of community members
+* Foster newcomers and develop their leadership potential
+* Moderate all official spaces where community members interact, including Discourse, Matrix, GitHub, and NixCon
+* Ensure that collaboration on shared technical goals is possible
+* Ensure that the Nix community is a welcoming space for people from marginalized backgrounds
+* Do not allow the Nix community to be a place for spreading ideas rooted in fascism or bigotry
```

Comment:
```suggestion
* Do not allow the Nix community to be a place for sharing ideas rooted in fascism or bigotry
```
I think this is more honest: "spreading ideas" has a negative connotation which brings legitimacy to the act of banning. "sharing ideas" has a positive connotation, which has the opposite effect (which I think is what we want): forcing us to think long and hard whether banning someone for sharing an opinion is something we want to do.